packages/web-components/src/components/link-list/link-list.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { html, customElement, property, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './link-list.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Link list.
+ *
+ * @element dds-link-list
+ */
+@customElement(`${ddsPrefix}-link-list`)
+class DDSLinkList extends LitElement {
+  @property({ type: String })
+  type = 'default';
+
+  // eslint-disable-next-line class-methods-use-this
+  protected render() {
+    return html`
+      <div class="${prefix}--link-list" data-autoid="${(this.constructor as typeof DDSLinkList).stableSelector}">
```

Comment:
To reduce extra DOM element created by `<dds-link-list-item>`, would you add the `:host(dds-link-list-item)` to the selector that styles `.bx--link-list`? Please see https://github.com/carbon-design-system/ibm-dotcom-library/blob/93db39f/packages/web-components/src/components/footer/footer-nav-item.ts#L40-L42 and https://github.com/carbon-design-system/ibm-dotcom-library/blob/93db39f3a3c304b0b064b37c3df6efe12fc1e429/packages/styles/scss/components/footer/_footer-nav-group.scss#L63-L90 as the reference. In this way, `.render()` has to render only the `<slot></slot>`. Thanks!